data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1023 @@
+			`	You see before you a couple hundred pirates, held captive by a large crowd of heavily-armed civilians - all of which are now hesitantly pointing guns at the operatives. It seems the citizens have arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians walk towards the operatives: one Hai, one human, and one Korath. The leader of the Republic operatives exits her suit and approaches them.`
+			`	The Hai speaks. "We are the leaders of the civilian government of Philosopher's Paradise. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We have heard of your war with the Free Worlds. Philosopher's Paradise is an independent nation; we will never join the Republic. I can, however, promise our country will turn to peace, and we will send no military ships through the wormhole."`
+			`	After several hours of discussion, the operatives leave their suits, preparing to return peacefully to Republic space.`
+			choice
+				`	(Chat with the Philosopher's Paradise citizens before you go.)`
+				`	(It's been a long day. Leave for your ship.)`
+					goto leave
+			`	You approach the human, Hai, and Korath who seem to represent the people of Philosopher's Paradise. The human turns to you and says, "You look like an asker of questions. What do you want to know?"`
+				goto "actual questions"
+			label questions
+			`	"Do you have more questions?"`
+			label "actual questions"
+			choice
+				`	"Do you want me to take you home?"`
+					goto home
+				`	"Do you know anything about the microtransceivers?"`
+					goto "microtransceivers"
+				`	"What can you tell me about this city?"`
+					goto place
+				`	"What do you do here?"`
+					goto do
+				`	"What do you know about this area of space?"`
+					goto space
+				`	"How did the Korath get here?"`
+					goto korath
+				`	"Can you speak to other Korath for me?"`
+					goto "translation services"
+				`	"Why were there Korath Raiders in this system?"`
+					goto korath
+				`	"I have no more questions."`
+			`	"It was a pleasure meeting you," says the human leader.`
+				goto leave
+			label korath
+			`	The Korath leader begins speak, with the Hai translating. "A few years ago, three of our raiding ships jumped into the Alcyone system. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship while us on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew.  The death is excruciatingly painful to Korath. Not wanting that fate, the rest of the raiders on the second and third Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not scientists like the others here, but we help as we can.`
```

Comment:
```suggestion
			`	The Korath leader begins speak, with the Hai translating. "Several generations ago, three of our raiding ships jumped into the Alcyone system. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship while us on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew.  The death is excruciatingly painful to Korath. Not wanting that fate, most of the raiders on the second and third Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not scientists like the others here, but we help as we can.`
```

Reply:
This may also be a good place to incorporate a reference to the short lifespan/generation time of the Korath?